### ISSUE TITLE
less encode/decode during replication

### ISSUE THREAD

**Issue by tsloughter** [SpaceTime-IoT/vonnegut#110]
I think we should be able to not have to decode and encode the record sets when replicating and write repairing.

**Comment by evanmcc** [comment]
I agree.  I made a comment to that effect somewhere in the code.  I was thinking about making the change when simplifying the replication, but I had some concern that I no longer remember.  It definitely seems doable.
